src/lib/rnp.cpp

```diff
@@ -3282,6 +3282,51 @@ rnp_key_export_revocation(rnp_key_handle_t key,
     return ret;
 }
 
+rnp_result_t
+rnp_key_revoke(
+  rnp_key_handle_t key, uint32_t flags, const char *hash, const char *code, const char *reason)
+{
+    if (!key || !key->ffi) {
+        return RNP_ERROR_NULL_POINTER;
+    }
+    if (flags) {
+        return RNP_ERROR_BAD_PARAMETERS;
```

Comment:
I almost feel like we should be using `RNP_ERROR_NOT_IMPLEMENTED` for this case, but the cat is pretty much out of the bag on that since we do it this way elsewhere.